kubernetes/.swagger-codegen-ignore

```diff
@@ -1,8 +1,8 @@
 .gitignore
 git_push.sh
-# Remove this once swagger-codegen 2.2.3 is released and we update. 
+# Remove this once swagger-codegen 2.3.0 is released and we update. 
```

Comment:
HEAD of swagger-codegen is technically still behind 2.3.0 since it hasn't been released yet. I'm realizing the line below I left filenames commented-out that shouldn't be overwritten until that PR is merged.